### DISCUSSION TITLE
Principal "dario" not in the set of valid principals for given certificate: ["ubuntu" "-teleport-internal-join"]] event:auth login:dario success:false

### DISCUSSION THREAD

**Discussion by darox** [gravitational/teleport#18418] [Q&A]
My teleport instance was working without any issues until I've added a new host with a different user name and now I'm getting the following error if I wanna connect to the hosts previously added with a different user name:
`Principal "dario" not in the set of valid principals for given certificate: ["ubuntu" "-teleport-internal-join"]] event:auth login:dario success:false`

What should I try  to resolve this issue?

**Comment by webvictim** [comment]
You may need to `tsh logout` and `tsh login` again to get updated certificates.

**Reply by darox** [reply]
I'm only using the web-ui. Is there something similar to tsh login / logout?

**Reply by webvictim** [reply]
Log out at the top right and log back in again.

**Reply by darox** [reply]
This has no effect, unfortunately.

**Reply by webvictim** [reply]
What is set for `logins` and `node_labels` in each of your roles?

**Reply by darox** [reply]
access:

```
    node_labels:
      '*': '*'
```

and 

```
    logins:
    - '{{internal.logins}}'
```

auditor:

```
logins:
    - no-login-a82cf323-2329-48c0-92c1-08bd6740bb90
```
and

no node label


editor:

no logins

and no node_labels

I haven't touched it so far.

**Reply by webvictim** [reply]
Try adding the login `dario` to the `access` role:

```
logins:
  - "{{internal.logins}}"
  - "dario"
```

Then log out and in again. You'll know it's worked if you see "dario" in the list of users when you click the "Connect" button next to the node.
